releasenotes/notes/adjust-symbolic-pulses-amp-angle-representation-f5c40007416cf938.yaml

```diff
@@ -0,0 +1,32 @@
+---
+other:
+  - |
+    Qiskit Terra 0.23.0, began phasing out support of complex parameters
+    in the Pulse module. Mainly, all library symbolic pulses were converted
+    from complex amplitude representation to a duo of real (float) parameters
+    (amp, angle). To avoid problems, Qiskit Experiments adopted this convention.
+    
+    Changes were made to :class:`.FixedFrequencyTransmon` and :class:`.HalfAngleCal`
+    (see upgrade section). With the exception of :class:`.HalfAngleCal`, all 
+    library experiments should continue function as they did before (even with 
+    complex ``amp``). When used with :class:`.FixedFrequencyTransmon` library, 
+    :class:`.HalfAngleCal` will also continue working as before.
+upgrade:
+  - |
+    The representation of pulses in the :class:`.FixedFrequencyTransmon` library 
+    was changed from complex amplitude to (amp,angle) representation. All pulses
+    now include an ``angle`` parameter, and the default values of ``amp`` are set
+    to float instead
+  - |
+    :class:`.HalfAngleCal`, was changed from updating the complex amplitude of 
+    the pulse, to updating the angle in the (amp, angle) representation. When used with
+    the :class:`.FixedFrequencyTransmon` library, it will continue to work seamlessly 
```

Comment:
Should we note here or in the other note that there is some awkward behavior when loading stored calibration parameters? I think if you use the new library, everything will work (with some deprecation warnings), but you will have loaded a complex amplitude and the half angle cal will start modifying the new `angle` parameter and leave the amplitude with whatever complex angle it was loaded with.

Reply:
Good catch. I think it's worth noting.

Reply:
Thank you.